src/components/global/Playground/playground.css

```diff
@@ -0,0 +1,107 @@
+/* Dark theme overrides */
+[data-theme="dark"] .playground {
+  --playground-btn-color: var(--c-carbon-20);
+  --playground-btn-selected-color: #fff;
+  --playground-btn-selected-background: var(--c-carbon-70);
+
+  --playground-separator-color: var(--c-carbon-70);
+}
+
+.playground {
```

Comment:
Can we move the variable documentation into one block, similar to how the components do it? Would be a bit easier to read.